scanner/Makefile

```diff
@@ -41,6 +41,12 @@ endif
 # The namespace to use for targets that require it (E2E, certs, etc).
 SCANNER_NAMESPACE ?= stackrox
 
+# The init bundle version to use when building the db image.
+SCANNER_IMAGE_DB_INIT_BUNDLE_VERSION ?= dev
```

Comment:
This will likely disappear and be replaced by a Makefile target like `make image-db-vulns`, which downloads the correct vulnerability bundle based on the current tag in the repo. The workflow then will checkout the correct tag, call that make target, and run the updater to create bundles using that -- one init bundle for each release based on the vuln data created by the export job.